### PR TITLE
Assingment_Q4_Prachi

### PR DESCRIPTION
**Pull request by PrachiJainTR**
WAP to find the area and perimeter of Rectangle & Circle

### CODE REVIEW

File: JavaAssignment/src/prachiJ/AreaPerimeterFind.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package prachiJ;
+
+import prachiJ.AreaPerimeterFind;
+
+public class AreaPerimeterFind {
+
+	void rectangle(float a,float b)
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/prachiJ/AreaPerimeterFind.java]: Using this method name I am not getting what are you going to perform in this method. So here name should be like areaOfRectangle, Right ?

Do this change for others methods.

File: JavaAssignment/src/prachiJ/AreaPerimeterFind.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		System.out.println("Perimeter is: " + perimeter);
+		System.out.println("Area is: " + area);
+	}
+	
+	
+	public static void main(String[] args) {
+		float a = Float.parseFloat(args[0]);
+		float b = Float.parseFloat(args[1]);
+		float r = Float.parseFloat(args[2]);
+
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/prachiJ/AreaPerimeterFind.java]: Remove extra space and add comments before your method and class.

File: JavaAssignment/src/prachiJ/AreaPerimeterFind.java (hunk 1 above)
Comment (PrachiJainTR) [reply JavaAssignment/src/prachiJ/AreaPerimeterFind.java]: Thank you Krishna for the feedback. 
I have have added comments, changed method name and did formating.

**Review by mkanani13 — APPROVED** [review]
code looks good to me 
keep practicing.

**Pull request merged**